modelci/types/models/profile_results.py

```diff
@@ -0,0 +1,90 @@
+import ipaddress
+from datetime import datetime
+from typing import Any
+
+from pydantic import BaseModel
+
+from modelci.types.models.common import InfoTuple
+
+
+class ProfileMemory(BaseModel):
+    """Memory class in dynamic profiling result business object.
+
+    Args:
+        total_memory (int): Total memory in Byte.
+        memory_usage (int): GPU memory used in Byte.
+        utilization (float): GPU utilization.
+    """
+
+    total_memory: int
+    memory_usage: int
+    utilization: float
+
+
+class ProfileLatency(BaseModel):
+    """End-to-end latency in dynamic profiling result business object.
+
+    This class records the end-to-end latency in four stages: data engine initialization, pre-processing, inference, and
+    post-processing.
+
+    TODO: support Iterable args, refer to dynamic_profile_result_bo.py
+
+    Args:
+        init_latency (InfoTuple): initialization latency.
+        preprocess_latency (InfoTuple): pre-processing latency.
+        inference_latency (InfoTuple): inference latency.
+        postprocess_latency (InfoTuple): post-processing latency.
+    """
+    #init_latency: InfoTuple = InfoTuple(avg=0, p50=0, p95=0, p99=0)
+    #preprocess_latency: InfoTuple = InfoTuple(avg=0, p50=0, p95=0, p99=0)
```

Comment:
Why do you comment this?

Reply:
Because these have not been implemented in `metric.py` yet. So I comment these to make the output results neater.

Reply:
You may keep this, as this shall be stored in the database. Otherwise, there will be a display error in CLI after fetching the profiling result.